com.dynamo.cr/com.dynamo.cr.bob/scripts/copy.sh

```diff
@@ -56,7 +56,7 @@ cp -v $DYNAMO_HOME/ext/bin/x86_64-win32/spirv-cross.exe libexec/x86_64-win32/spi
 #
 cp -v $DYNAMO_HOME/archive/${SHA1}/engine/share/builtins.zip lib/builtins.zip
 
-cp -v $DYNAMO_HOME/archive/${SHA1}/engine/armv7-android/classes.dex lib/classes.dex
+cp -v $DYNAMO_HOME/archive/${SHA1}/engine/arm64-android/classes.dex lib/classes.dex
```

Comment:
I don't download armv7 folderw anymore, so we can take this file from arm64 instead (it's the same file idk why we upload both, I don't wanna spend time fixing it here)